index.html

```diff
@@ -7,31 +7,40 @@
     <meta name="description" content="Abdul Ali Noori's portfolio" />
     <link rel="stylesheet" href="styles.css">
     <link rel="stylesheet" href="desktop-styles.css">
+    
     <title>Portfolio</title>
 </head>
 <body>
     <div class="wrapper">
         <header>
             <nav class="navbar">
-
-                <a href="#headline" class="welcome-logo">Noori</a>
-
-                <ul class="menu">
-                    <li><a class="menuItem" href="#">Portfolio</a></li>
-                    <li><a class="menuItem" href="#">Abbout</a></li>
-                    <li><a class="menuItem" href="#">Contact</a></li>
+    
+                <a href="#" class="nav-branding">Noori</a>
+    
+                <ul class="nav-menu">
+    
+                    <li class="nav-item">
+                        <a class="nav-link" href="#recent-work">Portfolio</a>
+                    </li>
+    
+                    <li class="nav-item">
+                        <a class="nav-link" href="#about-me">About</a>
+                    </li>
+                    
+                    <li class="nav-item">
+                        <a class="nav-link" href="#from-section">Contact</a>
+                    </li>
+    
                 </ul>
-
-                <button class="hamburger">
-                    <a class="menuIcon" href="">
-                        <img width="30px" src="images/Icons/Icon-Menu.svg" alt="Humberger menu">
-                    </a>
-                    <a class="closeIcon" href="">
-                        <img width="30px" src="images/Icons/Icon-close.svg" alt="close menu">
-                    </a>
-                </button>
-
+    
+                <div class="humburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
```

Comment:
- [ ] Kindly😃try to use the correct color of the Figma to the ```X``` icon to be like the Figma Design😊
- [ ] Also consider😊 decreasing the size of the icon to be like the Figma Design and with that, it will be nice for the visitors👌

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/217323719-dc5929ff-5243-42e5-8606-9865c6c6f66d.png)   |   ![image](https://user-images.githubusercontent.com/83360636/217323810-512ddfd8-4abb-434e-b026-48bc5194779e.png) |